src/client/js/Panels/MetaEditor/MetaEditorControl.js

```diff
@@ -1783,7 +1758,16 @@ define(['js/logger',
         if (WebGMEGlobal.State.get(CONSTANTS.STATE_ACTIVE_TAB) !== null &&
             WebGMEGlobal.State.get(CONSTANTS.STATE_ACTIVE_TAB) !== undefined &&
             metaAspectSheetsRegistry.length > WebGMEGlobal.State.get(CONSTANTS.STATE_ACTIVE_TAB)) {
-            selectedSheetID = WebGMEGlobal.State.get(CONSTANTS.STATE_ACTIVE_TAB);
+            //only the active panel should react to the global state
+            if (WebGMEGlobal.PanelManager._activePanel === this) {
+                selectedSheetID = WebGMEGlobal.State.get(CONSTANTS.STATE_ACTIVE_TAB);
+            } else {
+                for(selectedSheetID in this._sheets || {}){
```

Comment:
Style...

Looks good otherwise (thought you explicitly were parsing the URL when we talked in the kitchen).